docs/meshcloud.project.md

```diff
@@ -34,6 +34,8 @@ When adding a [meshPlatform](meshcloud.platform-location.md) to a meshProject, m
 
 You can configure one or multiple [payment methods](meshcloud.project-metering.md#payment-methods) and [billing addresses](meshcloud.project-metering.md#company-billing-addresses) for you meshCustomer. For a meshProject you have to select a specific one, so the project resources can be billed via a specific payment method and to a specific billing address.
 
+You also have the option of setting an alternate payment method on a project. This feature is enabled via a configuration in meshStack. A job that runs daily will check if the already assigned payment method is expired and if it is, will replace it with the alternate payment method.
```

Comment:
d: you can link this into the operator documentation.

Reply:
I don't understand what you mean. Do you mean add a link from operator documentation to this page?

Reply:
I will resolve and merge this. Will create a different PR when I have the answers.